File: largest_element.py
arr=[]
arrLen=int(input("Enter array size: "))
print(f"Enter {arrLen} elements")
for _ in range(arrLen):
  ele=int(input())
  arr.append(ele)
largeEle=arr[0]
for i in range(1,arrLen):
  if(arr[i]>largeEle):
    largeEle=arr[i]
print("Largest element: ",largeEle)

File: largest_element.c
#include<stdio.h>
int main()
{
  int arrLen, largeEle;
  printf("Enter array size: ");
  scanf("%d",&arrLen);
  int arr[arrLen];
  printf("Enter %d elements: ",arrLen);
  for (int i=0; i<arrLen; i++){
    scanf("%d",&arr[i]);
  }
  largeEle=arr[0];
  for (int j=1; j<arrLen; j++){
    if(arr[j]>largeEle){
      largeEle=arr[j];
    }
  }
  printf("Largest element: %d",largeEle);
  return 0;
}